lib/rules/web/admin_console/4.2/node.xyaml

```diff
@@ -0,0 +1,91 @@
+add_taint_to_node:
+  action: goto_one_node_page
+  action: click_key_and_action_link
+  action: click_add_more_button
+  action: add_taint
+  action: submit_changes
+add_tolerations_to_pod:
+  action:
+    if_param: dc_name
+    ref: goto_one_dc_page
+  action:
+    if_param: deploy_name
+    ref: goto_one_deployment_page
+  action: click_key_and_action_link
+  action: click_add_more_button
+  action: add_tolerations
+  action: submit_changes
+remove_tolerations_from_pod:
+  action:
+    if_param: dc_name
+    ref: goto_one_dc_page
+  action:
+    if_param: deploy_name
+    ref: goto_one_deployment_page
+  action: click_key_and_action_link
+  action: delete_node_affinity
+  action: submit_changes
+remove_taint_from_node:
+  action: goto_one_node_page
+  action: click_key_and_action_link
+  action: delete_node_affinity
+  action: submit_changes
+node_affinity_modal_value:
+  element: 
+    selector: &value
+      xpath: //div[text()='Value']/following-sibling::input
+node_affinity_modal_key:
+  element: 
+    selector: &key
+      xpath: //div[text()='Key']/following-sibling::input
+node_affinity_modal_dropdown:
```

Comment:
Is this action used?

Reply:
OK I see, this action is only used to define an anchor. Then I have the same concern/comments as below, it can be updated as `cick_node_affinity_modal_dropdown` since it's always called with a `click` action